ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/ShelleyHFC.hs

```diff
@@ -405,26 +405,27 @@ instance (SOP.All ShelleyBasedEra eras, Typeable eras) => ToCBOR (ShelleyTxOut e
     where
       each ::
            ShelleyBasedEra era
-        => SOP.Index eras era03
+        => SOP.Index eras era
         -> TxOutWrapper era
         -> SOP.K CBOR.Encoding era
       each idx (TxOutWrapper txout) = SOP.K $
-           CBOR.encodeListLen 2 <> CBOR.encodeTag (toEnum (idxLength idx))
+           CBOR.encodeListLen 2
+        <> CBOR.encodeWord (toEnum (idxLength idx))
         <> toCBOR txout
 
 instance (SOP.All ShelleyBasedEra eras, Typeable eras) => FromCBOR (ShelleyTxOut eras) where
   fromCBOR = do
       CBOR.decodeListLenOf 2
-      tag <- CBOR.decodeTag
+      i <- CBOR.decodeWord
```

Comment:
For what it's worth, we do call this value `tag` throughout the codebase (I think), despite not using `encodeTag`. It is "a tag", just not a "CBOR Tag" :/

Reply:
Reverted :+1: